Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:22.14
+
+# set git sha and docker tag form build time arg to run time env in container
+ARG GIT_SHA
+ARG DOCKER_CHANNEL
+ENV GIT_COMMIT=$GIT_SHA
+ENV DOCKER_TAG=$DOCKER_CHANNEL
+
+WORKDIR /price-pusher
+ADD . .
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)